fe/fe-core/src/main/java/com/starrocks/system/IComputable.java

```diff
@@ -0,0 +1,8 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.system;
+
+public interface IComputable {
+
+    boolean handleHbResponse(BackendHbResponse hbResponse);
```

Comment:
handleHbResponse method as  IComputable interface strange. If we don't know what interfaces IComputable should have. we could make it as a empty interface and add todo here.

For a important new interface, we should add some comments.

Reply:
Should explain why we need this interface